### DISCUSSION TITLE
Use rule Output in expression of another rule

### DISCUSSION THREAD

**Discussion by djrerun** [microsoft/RulesEngine#385] [Q&A]
Is it possible to reference the Output of a earlier executed rule in the expression of a later executed rule? If so, is there an example provided in the documentation?

**Comment by uribusoft** [comment]
This feature is very important to us as well , if there was a way to update the success event from inside the rule or just add a result ( number or object that is a result of an expression  ) to the rule result , we need to evaluate an expression and use the result numer that gets evaluated for the next rule .

**Comment by abbasc52** [comment]
@djrerun It is possible via evaluateRule and using `ExecuteActionWorkflowAsync` method
EvaluteRule allows you to define which rule to call next and also allows you to pass additional inputs / filter inputs

You can find the details here - https://microsoft.github.io/RulesEngine/#usage-1

One differentiation is you will need to explicitly define the sequence of rules to execute
